web/src/test/java/org/apache/shiro/web/servlet/SimpleCookieTest.java

```diff
@@ -157,12 +157,27 @@ private static <T extends javax.servlet.http.Cookie> T eqCookie(final T in) {
         reportMatcher(new IArgumentMatcher() {
             public boolean matches(Object o) {
                 javax.servlet.http.Cookie c = (javax.servlet.http.Cookie) o;
-                return c.getName().equals(in.getName()) &&
```

Comment:
I am not sure I see the value of this either, but I can see if this is OK too, so not too opinionated on this.